src/main/resources/hudson/plugins/jobConfigHistory/JobConfigBadgeAction/badge.jelly

```diff
@@ -1,8 +1,8 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" >
   <j:if test="${it.showBadge() and it.oldConfigsExist()}">
-    <a href="${it.createLink()}" id="showDiff">
-      <img width="16" height="16" title="${it.getTooltip()}" alt="x" src="${resURL}${it.getIcon()}"/>
-    </a>
+    <span class="icon-sm" >
+      <l:icon id="showDiff" src="${it.getIcon()}" href="${it.createLink()}" title="${it.getTooltip()}" alt="x" />
+    </span>

```

Comment:
This doesn't quite work out, I can't click the icon, how about
```xml
<a href="${it.createLink()}">
    <span class="icon-sm">
      <l:icon src="${it.getIcon()}" title="${it.getTooltip()}" />
    </span>
</a>
```

Reply:
done